README.md

```diff
@@ -1,68 +1,43 @@
-# The Planetary Software Organization Technical Steering Committee
+# The Planetary Software Organization
 
-The Planetary Software Organization Technical Steering Committee (TSC) is the technical
-governing body of the Planetary Software Organization. It is described in the
-[TSC Charter][].
+The Planetary Software Organization promotes open source software in the
+planetary sciences by helping software creators and maintainers foster an
+open source community.
 
-The governing documents for the TSC are locating in the root folder of this repository.
+The Planetary Software Organization helps software creators grow
+and organize their community by providing a set of ready-to-use
+community guidelines, standards, and governance documents.  The TSC
+also helps software maintainers manage the growth and evolution of
+their open source software by providing mentorship and suggestions.
 
-## TSC Scope
 
-**The TSC is responsible for those projects which are under TSC stewardship, and provides responsive input and organizational help to any projects which are not under TSC stewardship but which qualify as top level projects (TLPs).**
+## The Planetary Software Organization Technical Steering Committee
 
-The TSC exercises autonomy in managing its responsibilities.
+The Technical Steering Committee (TSC) is the governing body of the
+Planetary Software Organization, as described in its
+[Charter](TSC-Charter.md).
 
+The TSC primarily provides responsive input and organizational help
+to its Top-Level Projects and Working Groups, but is also available
```

Comment:
although a project is self explanatory a definition of what a working group is might be worth adding

Reply:
There are sections below here that provide links that go to (or will go to) TLP and TLWG documents, which I thought was enough at this level.